notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb

```diff
@@ -0,0 +1,835 @@
+{
```

Comment:
<p>I think it would be interesting to give a brief overview of each of the algos we are using</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/687/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Good point.

Reply:
Ah.. nvm Miguel's talking about the algos used in NNI. I was more like wonder about when to use hyperopt vs hyperdrive vs NNI